internal/serve/serve.go

```diff
@@ -359,30 +243,26 @@ func handler(deps handlerDeps) http.Handler {
 		})
 
 		r.Route("/tx", func(r chi.Router) {
-			handler := &httphandler.AccountHandler{
+			accountHandler := &httphandler.AccountHandler{
 				AccountService:            deps.AccountService,
 				AccountSponsorshipService: deps.AccountSponsorshipService,
 				SupportedAssets:           deps.SupportedAssets,
 				AppTracker:                deps.AppTracker,
 			}
 
-			r.Post("/create-sponsored-account", handler.SponsorAccountCreation)
-			r.Post("/create-fee-bump", handler.CreateFeeBumpTransaction)
+			r.Post("/create-sponsored-account", accountHandler.SponsorAccountCreation)
+			r.Post("/create-fee-bump", accountHandler.CreateFeeBumpTransaction)
 		})
 
-		r.Route("/tss", func(r chi.Router) {
-			handler := &httphandler.TSSHandler{
-				Router:             deps.TSSRouter,
-				Store:              deps.TSSStore,
+		r.Route("/transactions", func(r chi.Router) {
+			handler := &httphandler.TransactionsHandler{
+				TransactionService: deps.TransactionService,
```

Comment:
I think the prefix are redundant here:
- /tx/create-sponsored-account
- /tx/create-fee-bump
- /transactions/build

I think building and fee bumping are closely related and should likely be together.

In terms of creating a sponsored account, maybe that could move under `/accounts/...`. WDYT?

Considering the PR is already pretty big, these changes could be done in a follow-up PR.